src/Benchmark/Program.cs

```diff
@@ -1,14 +1,34 @@
+using System.Linq;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Jobs;
 using BenchmarkDotNet.Running;
+using RunMode = BenchmarkDotNet.Jobs.RunMode;
 
 namespace Benchmark
 {
+    public class FastAndDirty : ManualConfig
+    {
+        public FastAndDirty()
+        {
+            var job = new Job("", RunMode.Short, InfrastructureMode.InProcess);
```

Comment:
I _think_ this could be simplified to just this so you don't need to add all the loggers etc. in.

```csharp
public FastAndDirty()
{
    var job = Job.Core
        .With(InProcessToolchain.Instance)
        .WithId("InProcess");

    Add(job);
    Add(MemoryDiagnoser.Default);
}
```